updateBaseImages.sh

```diff
@@ -0,0 +1,311 @@
+#!/bin/bash -e
```

Comment:
why are you including this file if you're curling it from crw repo?

Reply:
it shouldn't be there :smile: